### ISSUE TITLE
Mind Map Issues/Epics

### ISSUE THREAD

**Issue by DaveMBush** [ZenHubIO/support#862]
I've been playing with the process of breaking out Epics to Stories and Stories to Task and I find that using a Mind Map to do this makes the process a lot easier.  I've found a few places on the web that talk about doing this as part of the refinement process but I've yet to find a Kanban tool that actually integrates mind maps and kanban in any reasonable way.

**Comment by azenMatt** [comment]
Thanks @DaveMBush - that could be very interesting

We have a number of updates coming to epics over the next few releases, this one is probably a bit more complicated to implement to make that, but we will look into it for future releases. 

One change we are looking at that might make the current process a little easier, is tagging the children of epics with a label so you can easily see what is an epic, and what is a child of an epic

**Comment by DaveMBush** [comment]
Yes, I realize it is probably a bit more advanced and probably needs quite a bit of grooming before you even start coding it.

Dave Bush
Contract Programmer/Architect/Mentor
JavaScript/ASP.NET/Node.JS
DMB Consulting, LLC
203.214.1758
http://blog.dmbcllc.com<http://blog.dmbcllc.com/>
https://www.linkedin.com/in/davembush
https://github.com/DaveMBush
http://stackoverflow.com/users/1876155/dave-bush

**Comment by nickbaldwin** [comment]
Probably the best way to approach this is storymapping (see http://jpattonassociates.com/wp-content/uploads/2015/03/story_mapping.pdf) - great agile approach to planning ideas - and then which become Areas, Epics, Stories  (in storymapping they use the name activities, tasks and stories)... anyway - it is a great approach to planning - and would be a fantastic view to add to bring the rest of the organization (particularly product managers) into this world.

There are some tools that do this e.g. https://realtimeboard.com/examples/user-story-map/ but the only simple one that is integrated in a decent way is an addon for JIRA: https://marketplace.atlassian.com/plugins/com.arijea.easy-agile-roadmaps/cloud/overview
 & https://marketplace.atlassian.com/plugins/com.kretar.jira.plugin.user-story-map/cloud/overview

Nobody provides that for github. 

Even just providing swimlanes - or rows - like https://overv.io/ would be good though - having that second dimension really helps with planning.
